charts/workload-identity-webhook/values.yaml

```diff
@@ -2,14 +2,12 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-labels:
-  azure-workload-identity.io/system: "true"
```

Comment:
What was this again?

Reply:
The labels value was never configured in helm chart and we replaced it with `podLabels` (ref:  https://github.com/Azure/azure-workload-identity/pull/627#discussion_r1016173935)